contracts/modules/common/BaseModule.sol

```diff
@@ -83,4 +83,26 @@ contract BaseModule is Module {
     function isOwner(BaseWallet _wallet, address _addr) internal view returns (bool) {
         return _wallet.owner() == _addr;
     }
+
+    /**
+     * @dev Helper method to invoke a wallet.
+     * @param _wallet The target wallet.
+     * @param _to The target address for the transaction.
+     * @param _value The value of the transaction.
+     * @param _data The data of the transaction.
+     */
+    function invokeWallet(address _wallet, address _to, uint256 _value, bytes memory _data) internal returns (bytes memory _res) {
+        bool success;
+        // solium-disable-next-line security/no-call-value
+        (success, _res) = _wallet.call.value(0)(
+            abi.encodeWithSignature("invoke(address,uint256,bytes)", _to, _value, _data)
+        );
+        if(success && _res.length > 0) { //_res is empty if _wallet is an "old" BaseWallet that can't return output values
+            (_res) = abi.decode(_res, (bytes));
+        } else if (_res.length > 0) {
+            revert(string(_res));
+        } else if(!success) {
```

Comment:
We need it because we need to make sure we are not in the 4th possibility (`success == true && _res.length == 0`) for which we do not want to do anything

Reply:
Indeed, 2 conditions makes 4 possibilities :-)